document/oleobject.go

```diff
@@ -0,0 +1,54 @@
+//name: zhexiao(肖哲)
```

Comment:
https://github.com/unidoc/unioffice/wiki/UniOffice-Developer-Guide#7-license-header-in-each-go-file

Reply:
Fixed.